theme/theme.inc

```diff
@@ -60,6 +60,11 @@ function islandora_preprocess_islandora_default_edit(array &$variables) {
           'datastream' => $ds,
         )),
       );
+    }
+    else if (user_access(ISLANDORA_VIEW_DATASTREAM_HISTORY)) {
```

Comment:
@jonathangreen i think @jordandukart is ok with the logic, it was more about the "else if" with a space in between.  👍